model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,158 @@
+groups:
+  - id: registry.cloudfoundry
+    prefix: cloudfoundry
+    type: attribute_group
+    brief: >
+      CloudFoundry resource attributes.
+    attributes:
+      - id: system.id
+        type: string
+        stability: experimental
+        brief: >
+          A guid or another name describing the event source.
+        note: |
+          CloudFoundry defines the `source_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the component name, e.g. "gorouter", for
+          CloudFoundry components.
+
+          When system components are instrumented, values from the
+          [Bosh spec](https://bosh.io/docs/jobs/#properties-spec)
+          should be used. The `system.id` should be set to
+          `spec.deployment/spec.name`.
+        examples:
+          - 'cf/gorouter'
+      - id: system.instance.id
+        type: string
+        stability: experimental
+        brief: >
+          A guid describing the concrete instance of the event source.
+        note: |
+          CloudFoundry defines the `instance_id` in the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is used for logs and metrics emitted by CloudFoundry. It is
+          supposed to contain the vm id for CloudFoundry components.
+
+          When system components are instrumented, values from the
+          [Bosh spec](https://bosh.io/docs/jobs/#properties-spec)
+          should be used. The `system.instance.id` should be set to `spec.id`.
+        examples:
+          - '218fc5a9-a5f1-4b54-aa05-46717d0ab26d'
+      - id: app.name
```

Comment:
is it similar to the `service.name` ?

Reply:
Just to summarise the arguments of the ongoing conversation: For CF applications, it is very likely, that this `app.name` should also be used as the `service.name`. There might be deviations from that, e.g., including app instance index or org and space names. Furthermore, the `service.name` might be completely different for other reasons. Therefore, I suggest to provide a recommendation to use the `app.name` for the `service.name` but allow for a different value. To save data volume, the `app.name` can be omitted. This is actually true for all `*.name` attributes, if their `*.id` counter parts are there and vice versa. 

@lmolkova would you be alright with keeping this field as optional and adding the recommendation for the service name?